src/main/java/io/pivotal/literx/Part06Request.java

```diff
@@ -38,7 +38,7 @@ StepVerifier requestOneExpectSkylerThenRequestOneExpectJesse(Flux<User> flux) {
 
 //========================================================================================
 
-	// TODO Return a Flux with all users stored in the repository that prints "Starring:" on subscribe, "firstname lastname" for all values and "The end!" on complete
+	// TODO Return a Flux with all users stored in the repository that prints "Starting:" on subscribe, "firstname lastname" for all values and "The end!" on complete
```

Comment:
This is not a typo: `Starring`, as "a movie starring user1, user2, user3, ..." (see the `solution` branch). That said, the associated test doesn't check the console output, so it is not terribly meaningful.